src/assets/wise5/common/Node.spec.ts

```diff
@@ -28,6 +27,24 @@ function copyComponents() {
   });
 }
 
+function resetComponents() {
+  node = Object.assign(new Node(), {
+    components: [{ id: componentId1 }, { id: componentId2 }]
+  });
```

Comment:
This code is duplicated in the insertComponents() function in the same file. There are also other places in the file where the components are initialized so the resetComponents() function name seems a little inaccurate. Maybe the top describe function should have a beforeEach() and inside of it the node should be initialized to new Node(). Then in the test functions they can set node.components to whatever they need.

Reply:
Good point. I made some changes. PTAL